addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -73,6 +73,12 @@ WysiwygMultizone.include({
      */
     start: function () {
         $('.js_tweet, .js_comment').off('mouseup').trigger('mousedown');
+        _.each($('.oe_structure[data-editor-message!="False"]'), function (el) {
+            if(!$.trim($(el).html()).length) {
+                $(el).empty();
+            }
+            $(el).toggleClass('oe_empty', !$.trim($(el).html()).length);
+        });
```

Comment:
Probably useless code ? `oe_empty` is not something that should or not be there, it signals it **can** be empty.